### PR TITLE
Website: replace broken referral link to TEALS volunteer program

### PR DESCRIPTION
**Pull request by tanyaparker**

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/help.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ Code.org's online platform [Code Studio](https://studio.code.org) enables studen
 
 [col-66]
 
-- [TEALS](http://c.tealsk12.org/l/667) brings together hundreds of software engineers and teachers to help build sustainable CS classes across the country. Join the more than 1,500 CS professionals who have made an impact in local high schools by volunteering 2 days a week to teacher Intro and AP CS classes.
+- [TEALS](http://r.tealsk12.org/tbz7xx) brings together hundreds of software engineers and teachers to help build sustainable CS classes across the country. Join the more than 1,500 CS professionals who have made an impact in local high schools by volunteering 2 days a week to teacher Intro and AP CS classes.
```
Comment (breville) [inline pegasus/sites.v3/code.org/public/help.md:66]: Is this a permanent link?

Comment (tanyaparker) [reply pegasus/sites.v3/code.org/public/help.md:66]: Permanent replacement?

Comment (breville) [reply pegasus/sites.v3/code.org/public/help.md:66]: I mean are they maintaining that link long-term?  It just feels kind of temporary :)

Comment (tanyaparker) [reply pegasus/sites.v3/code.org/public/help.md:66]: I guess I didn't ask what happened to the last one :/

**Pull request merged**